ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -130,14 +136,44 @@ private T GetSetting<T>(string key, T defaultValue)
                     return (T)data;
                 }
             }
+#else
+            if (typeof(T) == typeof(bool))
+            {
+                try
+                {
+                    Boolean result = Boolean.TryParse(ConfigurationManager.AppSettings[key], out result);
+                    return (T) (object) result;
+                }
+                catch (ConfigurationErrorsException)
+                {
+
+                }
```

Comment:
Is there nothing that needs to be done if there's a ConfigurationError?  Shouldn't you just let this exception through?